Elevato/Elevato.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
```

Comment:
what is elevato? this is a poor name I dont know what it means or does

Reply:
this is fixed